microservice/frontend/catalog/debian/data/etc/nginx/sites-available/shop.microservice.io

```diff
@@ -1,3 +1,17 @@
+
+upstream _backend_old {
+  server 10.211.55.104:81;
```

Comment:
Can you check if this works with FQDNs? (what you find in /etc/hosts). Those address will change when we go to OpenStack.